### ISSUE TITLE
Graphic Equalizer. More than one "Personal Preset"?

### ISSUE THREAD

**Issue by unoukujou** [Pittvandewitt/Wavelet#259]
One thing that's driving me crazy is that I have a couple of different headphones and once I EQ a pair of headphones the way I want, I save the Personal Preset. But then when I use another pair I have to EQ that one and it overwrites the Personal Preset that I spent all day EQ'ing for the other headphone. Then when I use those other headphones I have to EQ them all over again and it's  never ending cycle of having to re-EQ each pair every time.

Please allow us to save our own named presets. Then I can have one saved for each pair of headphones/speakers that I use. This is about the Graphic Equalizer part, not the AutoEQ.

**Comment by unoukujou** [comment]
Just want to add that this is happening when I use wired 3.5mm headphones. Not sure if it applies to Bluetooth connected headphones.

**Comment by WhitespaceQ** [comment]
I think this is part of the how the app were designed, after the sound passes AutoEQ the audio devices should have "similiar" sound signature and then after that you tune the "netural" sound to your preference (wheter the AutoEQ'd sound is truly neutra and nearly identical is a whole can of worms that's irrelevant here, but that's the spirit at least in my opinion).

But having a personalized named preset is a good addition, I can see this being a premium feature to increase the value of the current pro version. I don't see how the Android system can recognize different wired headphones tho so its still a manual step to change your preset.

**Comment by unoukujou** [comment]
Yeah, I only use wired headphones. Manually saving/loading named presets would solve a huge problem. For now I'm using another EQ app that has avolity to save my own presets. It's just that Wavelet is unusable for me without this.

I don't use AutoEQ and some if my headphones aren't even listed there. I just use the GraphicEQ and tune until it sounds good to me. But without the ability to save my work, it's all for nothing.

**Comment by aspidxt** [comment]
Another usecase for "custom presets" is for using bt speaker in different locations. Same speaker should be tuned differently for playing inside a room, outdor, backpack, etc.
But it's better to use custom presets for all effects at once, same as auto presets for different bt devices

**Comment by marciozomb13** [comment]
I still hope developer implement this. Too simple to ignore.

**Comment by Pittvandewitt** [comment]
This will be in the next release.
